src-theme/package.json

```diff
@@ -13,20 +13,20 @@
     "@sveltejs/vite-plugin-svelte": "^3.0.2",
     "@tsconfig/svelte": "^5.0.2",
     "@types/dateformat": "^5.0.2",
-    "sass": "^1.70.0",
-    "svelte": "^4.2.10",
```

Comment:
Why did you downgrade Svelte?

Reply:
uhh oops

Reply:
Resolved in 3e7c077699b728f76ea92c312b4d289b5ae81621